release-team/README.md

```diff
@@ -185,3 +185,19 @@ is undefined.
 ### During "Security" releases
 For all unplanned or embargoed releases
 - Facilitate security releases following the under the [Security Release Process](security-release-process-documentation/security-release-process.md)
+
+---
+
+## Milestone Maintainers
+
+Across release cycles, one of the best signals for [issue triage](https://github.com/kubernetes/community/blob/master/contributors/guide/issue-triage.md#milestones) is whether or not an issue or PR is actually targeted for the current milestone.
+
+To maintain up-to-date status on milestone inclusion, we rely on a set of Milestone Maintainers (members of the `milestone-maintainers` GitHub team) to apply the appropriate labels to issues / PRs. This is facilitated by `/milestone` commands and bot automation.
+
+Each release cycle, the current Release Team should seek to manage membership to the aforementioned GitHub team.
+
+The `milestone-maintainers` GitHub team should contain:
+- At least 2 SIG leads (either SIG Chairs or SIG Technical Leads) with Maintainer access to manage membership to the GitHub team (for their SIG)
+- A set of SIG Milestone delegates (known SIG feature reviewers / approvers), at least 2, which will work to keep issues / PRs / [feature tracking issues](https://github.com/kubernetes/features/issues) up-to-date for the current milestone
+
+Active membership to the GitHub team requires sustained milestone triage. SIG leadership should strive to ensure the membership is maintained and reviewed during every release cycle.
```

Comment:
Also we should add a note here to prune the list when folks stop using their powers.

Reply:
@dims -- will clarify the language

Reply:
Agree this needs clarification.  If you're not actively willing to participate in issue/pr triage for a given release cycle, you shouldn't need access to these commands.  With great power comes great responsibility.

Reply:
Do we have any ability to actually pull this information?  I know I can't do it with devstats (there's a problem with adds-milestone events).